Xamarin.Forms.Core/RelativeLayout.cs

```diff
@@ -308,7 +347,10 @@ public void Add(View view, Expression<Func<double>> x = null, Expression<Func<do
 
 			public void Add(View view, Constraint xConstraint = null, Constraint yConstraint = null, Constraint widthConstraint = null, Constraint heightConstraint = null)
 			{
-				Parent.CreateBoundsFromConstraints(view, xConstraint, yConstraint, widthConstraint, heightConstraint);
+				RelativeLayout.SetXConstraint(view, xConstraint);
+				RelativeLayout.SetYConstraint(view, yConstraint);
+				RelativeLayout.SetWidthConstraint(view, widthConstraint);
+				RelativeLayout.SetHeightConstraint(view, heightConstraint);
```

Comment:
these 4 calls should be wrapped between a `BatchBegin()` and a `BatchCommit()` calls.